TypeCobol/Compiler/Diagnostics/CodeElementCheckers.cs

```diff
@@ -297,5 +297,32 @@ public static void OnCodeElement(AlterStatement statement, CodeElementsParser.Al
         }
     }
 
+    class CodeElementChecker
+    {
+        public static void OnCodeElement(CodeElement codeElement, bool debuggingMode)
+        {
+            if (debuggingMode)
+            {
+                // detect CodeElement with a mix of Debug and "Normal" lines in debugging mode
+                int consumedTokensCount = codeElement.ConsumedTokens.Count;
+                if (consumedTokensCount > 1)
+                {
+                    bool isDebug = false, isNoDebug = false;
+                    for (int i = 0; i < consumedTokensCount; i++)
+                    {
+                        bool isDebugType = char.ToLower(codeElement.ConsumedTokens[i].TokensLine.IndicatorChar) == 'd';
+                        isDebug |= isDebugType;
+                        isNoDebug |= !isDebugType;
+                        if (isDebug && isNoDebug)
+                        {
+                            DiagnosticUtils.AddError(codeElement, "In debugging mode, a statement cannot be across lines marked with debug and lines not marked debug.");
```

Comment:
I think the message "In debugging mode, a statement cannot span across lines marked with debug and lines not marked debug." sounds more natural.

Reply:
done
https://github.com/TypeCobolTeam/TypeCobol/tree/8adf8c367cfdcd4aaa32b91135b29bd72e31b660